src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/GenericComponent.tsx

```diff
@@ -207,7 +207,13 @@ export function GenericComponent(props: IGenericComponentProps) {
 
   const RenderComponent = components.find(
     (componentCandidate) => componentCandidate.name === props.type,
-  ).Tag;
+  );
+  if (!RenderComponent) {
```

Comment:
Would be great with a test to cover this scenario :)

Reply:
Done